tsconfig.json

```diff
@@ -1,11 +1,27 @@
 {
     "compilerOptions": {
-        "allowJs": true
+        "jsx": "react",
+        "target": "ESNext",
+        "module": "es2015",
+        "rootDir": "src",
+        "outDir": "build",
+        "allowJs": true,
+        "sourceMap": true,
+        "moduleResolution": "node",
+        "strictNullChecks": true
     },
+    "include": [
+        "typings/index.d.ts",
+        "src/**/*.ts",
+        "src/**/*.tsx"
+    ],
     "exclude": [
         "node_modules",
         "Pod/Assets",
         "Example/Build",
-        "externals/"
-    ]
+        "externals",
+        "build",
+        "index.ios.js"
+    ],
+    "compileOnSave": true
```

Comment:
this we want to eliminate via https://github.com/orta/systems-theory/issues/1 and https://github.com/facebook/react-native/issues/11932